lightly/cli/upload_cli.py

```diff
@@ -40,8 +40,12 @@ def _upload_cli(cfg, is_cli_call=True):
         print_as_warning('Please specify your access token.')
         cli_api_args_wrong = True
 
-    dataset_id_ok = dataset_id and len(dataset_id) > 0
-    new_dataset_name_ok = new_dataset_name and len(new_dataset_name) > 0
+    if dataset_id and len(dataset_id) > 0:
+        dataset_id_ok = True
+    else:
+        dataset_id_ok = False
+        del dataset_id
```

Comment:
what, why?

Reply:
I wanted to delete `dataset_id`, so that no one will use it anywhere after this point, as it is an empty string. This would have prevented the implementation of the error in the first place.

Reply:
then just add a comment and it's ok

Reply:
I did and reworked the if-clauses to distinguish the 4 possible cases better.